### PR TITLE
Added variabes into stub to ease automated filling

### PR DESCRIPTION
**Pull request by axelaris**
Using these variables it will be easier to create manifests even without any manual editing

### CODE REVIEW

**Review by Infra-Red — COMMENTED** [review]
@axelaris  Could you please apply this changes for other stubs too.

File: templates/stub.openstack.example.yml [hunk 1]
```diff
@@ -98,37 +98,38 @@ jobs:
...
-    - 10.0.1.10 - 10.0.1.40
+    - 192.168.1.96 - 192.168.1.120
 
 - name: elastic
   type: vip
   cloud_properties:
     security_groups:  # Specify your security groups used for 'elastic' network
-    - MY-GROUP1
-    - MY-GROUP2
+    - bosh
```
Comment (hannayurkevich) [inline templates/stub.openstack.example.yml:134]: Please mention the sg in the deploy doc.

File: templates/stub.openstack.example.yml [hunk 2]
```diff
@@ -98,37 +98,38 @@ jobs:
...
+    - logsearch
```
Comment (hannayurkevich) [inline templates/stub.openstack.example.yml:135]: Please mention the sg in the deploy doc.

**Review by hannayurkevich — CHANGES_REQUESTED** [review]
Please make the same changes in other stubs. And also add security groups description to the deploy deoc.

File: templates/stub.openstack.example.yml (hunk 1 above)
Comment (axelaris) [reply templates/stub.openstack.example.yml:134]: done

File: templates/stub.openstack.example.yml (hunk 2 above)
Comment (axelaris) [reply templates/stub.openstack.example.yml:135]: done

**Comment by hannayurkevich** [comment]
Thanks, @axelaris!

**Pull request merged**